go/client/cmd_simplefs.go

```diff
@@ -69,6 +69,12 @@ func makeSimpleFSPath(g *libkb.GlobalContext, path string) keybase1.Path {
 
 	path = filepath.ToSlash(filepath.Clean(path))
 
+	// Certain users seem to want to use SimpleFS on their
+	// mounted KBFS
+	if strings.HasPrefix(path, mountDir) {
+		return keybase1.NewPathWithKbfs(path[len(mountDir):])
+	}
```

Comment:
`keybase/[...]` fails the `IsAbs()` test on Windows, so we have to test it there too.

`mountDir` is a constant we define, so yes.

I added comments and made mountDir a constant, and addressed some lint errors.